route53.tf

```diff
@@ -32,23 +32,20 @@ resource "aws_route53_record" "server_AAAA" {
 resource "aws_route53_record" "private_PTR" {
   zone_id = var.private_reverse_zone_id
   name = format(
-    "%s.%s.%s.%s",
-    element(split(".", aws_instance.openvpn.private_ip), 3),
-    element(split(".", aws_instance.openvpn.private_ip), 2),
-    element(split(".", aws_instance.openvpn.private_ip), 1),
-    element(split(".", aws_instance.openvpn.private_ip), 0),
+    "%s",
```

Comment:
@felddy, while fixing this I realized that Terraform and/or AWS appears to Do The Right Thing if you specify just enough of the name to "fill in" the PTR record.  For example, if this record were for the IP `10.11.12.13`, then you could provide the entire record name (`13.12.11.10.in-addr.arpa.`) or just the last octet (`13`).  If you do the latter, then look at the corresponding Route53 record in the AWS console, you can see that the `.12.11.10.in-addr.arpa.` part of the name has been automatically added.  With the previous code the record was coming out as `13.12.11.10.12.11.10.in-addr.arpa.`

I thought this might be useful information to point out to you, since you are our DNS expert.  At a minimum it allows us to create PTR records more succinctly.

Reply:
Wow.  I amazed this worked before and equally amazed this works with the last octet alone.  Thanks!
🤯

Reply:
@jsf9k Your explanation in the comment above would make an excellent code comment to benefit future developers (and our own forgetful selves).